File: Esame_22/PYTHON/Corso.py

class Corso:
    #### Gli attributi della classe sono questi _attributo ####

    # COSTRUTTORE
    def __init__(self, codiceCorso, nomeCorso, docente):
        self._codiceCorso = codiceCorso
        self._nomeCorso = nomeCorso
        self._docente = docente
    # _ indica che sono attributi ad uso interno, che non dovrebbero essere usati per
    # l'accesso dall'esterno

    #  METODI

    def toString(self):
        return self._nomeCorso + "\t" + str(self._codiceCorso) + "\t" + \
               self._docente + "\t"
    # con \ mi fa andare a capo col codice senza problemi

File: Esame_22/JAVA/Corso.java
import java.util.*;

public class Corso {
	// Attributi
	protected int codiceCorso;
	// protected String tipo; --> per modellare il tipo, lo faccio nel main
	protected String nomeCorso;
	protected String docente;

	// Costruttore
	public Corso(int codiceCorso, String nomeCorso, String docente) {
		this.codiceCorso = codiceCorso;
		this.nomeCorso = nomeCorso;
		this.docente = docente;
	}

	// Metodi

	// PUNTO 3: toString
	public String toString() {
		return this.nomeCorso + "\t" + this.codiceCorso + "\t" + this.docente + "\t";
	}

} // fine classe
